src/Features/Blockcore.Features.Wallet/Api/Controllers/WalletRPCController.cs

```diff
@@ -120,15 +120,20 @@ public bool LockWallet()
 
         [ActionName("sendtoaddress")]
         [ActionDescription("Sends money to an address. Requires wallet to be unlocked using walletpassphrase.")]
-        public async Task<uint256> SendToAddressAsync(BitcoinAddress address, decimal amount, string commentTx, string commentDest)
+        public async Task<uint256> SendToAddressAsync(BitcoinAddress address, decimal amount, string commentTx, string commentDest, decimal fee = 0, bool isSegwit = false)
         {
-            WalletAccountReference account = this.GetWalletAccountReference();
+            if (fee <= 0)
+            {
+                fee = this.FullNode.Network.MinTxFee;
```

Comment:
This did not happen before, so this is a new change, right?

Reply:
Yeah, so here I'm defaulting to the min network fee, if 0 is passed.

Reply:
What if I want to use 0 in fee? That's not possible?

Reply:
(I understand that the miner / nodes might not accept it with zero fee, but if a miner/staker allows it, or they want to send transactions through their own staking node, then it could be a valid option to use 0 fee, couldn't it?)

Reply:
You can see that I changed to to if not null, you can still pass 0 and it will honor it.